### PR TITLE
feat: Record metric on every InvokeModelCommand

### LINKED ISSUE #227: Library metric on InvokeModelCommand
A metric is recorded  to indicate that the agent is instrumenting AWS Bedrock.  This will occur on every `InvokeModelCommand` and `InvokeModelWithResponseStreamCommand`.
`Nodejs/ML/Bedrock/<version>`

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Description

This pr records a metric on every`InvokeModelCommand`. 

## How to Test

Added a new versioned test

## Related Issues

Closes #227

### CODE REVIEW

File: tests/versioned/v3/bedrock-chat-completions.tap.js [hunk 1]
```diff
@@ -208,4 +210,21 @@ tap.afterEach(async (t) => {
       t.end()
     })
   })
+  tap.test(
```
Comment (jsumners-nr) [inline tests/versioned/v3/bedrock-chat-completions.tap.js:213]:
```suggestion
  })
  
  tap.test(
```

File: lib/v3/bedrock.js [hunk 2]
```diff
@@ -164,6 +167,7 @@ function getBedrockSpec({ config, commandName }, _shim, _original, _name, args)
 function bedrockMiddleware(shim, config, next, context) {
   const { commandName } = context
   if (commandName === 'InvokeModelCommand') {
+    TRACKING_METRIC = `Nodejs/ML/Bedrock/${shim.pkgVersion}`
```
Comment (bizob2828) [inline lib/v3/bedrock.js:170]: this is getting re-assigned on every call. you could just put it in the `init` function

**Review by bizob2828 — CHANGES_REQUESTED** [review]
overall looks good. one suggestion on where to assign the `TRACKING_METRIC`

File: lib/v3/bedrock.js (hunk 2 above)
Comment (svetlanabrennan) [reply lib/v3/bedrock.js]: Done

**Review by bizob2828 — APPROVED** [review]

**Review by jsumners-nr — APPROVED** [review]

**Pull request merged**